src/providers/journal/journal.spec.ts

```diff
@@ -0,0 +1,263 @@
+          route: 9,
+          showMe: 3,
+          tellMe: 8
+        },
+        slot: {
+          _slotId: 1002,
+          start: '2018-03-05T09:07:00+00:00',
+          duration: 30,
+          _testCentreId: 54321,
+          testCentreName: 'Example Test Centre'
+        },
+        booking: {
+          candidate: {
+            _candidateId: 100,
+            candidateName: {
+              title: 'Miss',
+              firstName: 'Kamil',
+              secondName: 'Bbb',
+              thirdName: 'Ccc',
+              lastName: 'Zeilinski'
+            },
+            driverNumber: 'AAAAA123456A78AA',
+            gender: 'Female',
+            candidateAddress: {
+              line1: '1 Some Street',
+              line2: 'Some place',
+              line3: 'Some area',
+              line4: 'Some town',
+              line5: 'Some county',
+              postcode: 'AA1 2BB'
+            },
+            primaryTelephone: '01234 567890',
+            secondaryTelephone: '04321 098765',
+            mobileTelephone: '07654 123456',
+            email: 'aaaa@bbbb.com',
+            prn: 11223344,
+            previousADITests: 2
+          },
+          application: {
+            _applicationId: '384 392 293 22',
+            _bookingSequence: '03',
+            _checkDigits: '01',
+            welshTest: false,
+            extendedTest: true,
+            progressiveAccess: false,
+            checkMarker: false
+          }
+        }
+      },
+      {
+        slot: {
+          _slotId: 1003,
+          start: '2018-03-05T10:14:00+00:00',
+          duration: 90,
+          vehicleSlotType: 'Voc90mins',
+          _testCentreId: 54321,
+          testCentreName: 'Example Test Centre'
+        },
+        booking: {
+          candidate: {
+            _candidateId: 100,
+            candidateName: {
+              title: 'Miss',
+              firstName: 'Matthew',
+              secondName: 'Bbb',
+              thirdName: 'Ccc',
+              lastName: 'Murry'
+            },
+            driverNumber: 'AAAAA123456A78AA',
+            gender: 'Female',
+            candidateAddress: {
+              line1: '1 Some Street',
+              line2: 'Some place',
+              line3: 'Some area',
+              line4: 'Some town',
+              line5: 'Some county',
+              postcode: 'AA1 2BB'
+            },
+            primaryTelephone: '01234 567890',
+            secondaryTelephone: '04321 098765',
+            mobileTelephone: '07654 123456',
+            email: 'aaaa@bbbb.com',
+            prn: 11223344,
+            previousADITests: 2
+          },
+          application: {
+            _applicationId: '384 392 293 22',
+            _bookingSequence: '01',
+            _checkDigits: '06',
+            welshTest: false,
+            extendedTest: true,
+            progressiveAccess: false,
+            checkMarker: false
+          }
+        }
+      }
+    ]
+  }
+};
+
+const transformedMockData = [
```

Comment:
do we have some types for this structure?

Reply:
This will be the data that comes from TARs, it probably comes in as an any

Reply:
Yes, there isn't a type for the data returned from tars but there is for what it returned from the provider after the data has been transformed. It should probably be added to the test. :\